apps/webapp/src/app/theme/use-theme.tsx

```diff
@@ -0,0 +1,39 @@
+import { createTheme, Theme } from '@mui/material';
+import { useThemeMode } from './use-theme-mode';
+
+export const useTheme = (): [
+  theme: Theme,
+  themeMode: { change: () => void }
+] => {
+  const [mode, themeModeChange] = useThemeMode();
+  const theme = createTheme(
+    {
+      palette: {
+        mode: localStorage.getItem('theme') === 'light' ? 'light' : 'dark',
+        ...(localStorage.getItem('theme') === 'light'
```

Comment:
hmm, mam kontekst i provider a jednak wyciągam detal implementacyjny z local storage?
ja to zupełnie inaczej rozumiałem.
mam useTheme które mi zwraca theme, mam useThemeMode które zwraca mode i seMode/toggleMode. to po co jeszcze dodatkowo zaglądać ręcznie do local storage?